### PR TITLE
obetsun election day solution

### PR DESCRIPTION
**Pull request by olegbet**

### CODE REVIEW

File: election-day/obetsun/election_day.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package electionday
+
+import "strconv"
+
+// NewVoteCounter returns a new vote counter with
+// a given number of inital votes.
+func NewVoteCounter(initialVotes int) *int {
+	var counter int
```
Comment (enarha) [inline election-day/obetsun/election_day.go:8]: Something is breaking the indentation in your IDE. I see it here, in the next function, on line 26 and I remember seeing this on your MRs on the service repo. Get it fixed or take it to a doctor :-).
And you do not need the extra var here, just return the address of initialVotes.
Other than that it's all nice!

File: election-day/obetsun/election_day.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+// NewElectionResult creates a new election result
+func NewElectionResult(candidateName string, votes int) *ElectionResult {
+	electionResult := ElectionResult{candidateName, votes}
+    return &electionResult
+}
+
+// DisplayResult creates a message with the result to be displayed
+func DisplayResult(result *ElectionResult) string {
+	return result.Name+" ("+strconv.Itoa(result.Votes)+")"
```
Comment (enarha) [inline election-day/obetsun/election_day.go:32]: ah, and here you can use fmt.Sprintf to construct the string, it's more elegant, `fmt.Sprintf("%s (%d)", result.Name, result.Votes)`.

File: election-day/obetsun/election_day.go (hunk 1 above)
Comment (olegbet) [reply election-day/obetsun/election_day.go:8]: I think the indentation here is from vim-go plugin I use, it always replaces spaces with tabs.

File: election-day/obetsun/election_day.go (hunk 2 above)
Comment (olegbet) [reply election-day/obetsun/election_day.go:32]: You're right, actually this exercise I did long ago, don't know why I did it this way.

**Pull request merged**